docs/BLTouch.md

```diff
@@ -1,50 +1,129 @@
 
-    PROBE
+If the above is looking good, it's time to test that the probe responds to commands from the
+firmware. First run `BLTOUCH_DEBUG COMMAND=pin_down` in your printer terminal. Verify that the pin
+moves down, and that the red LED on the probe turns off. If not, check your wiring and configuration
+again. Next issue a `BLTOUCH_DEBUG COMMAND=pin_up` and verify that the pin moves up, and that the
+red light turns on again. If it's flashing then there's some problem.
 
-This should deploy the BL-Touch probe, stop the run when it triggers by touching the bed.
-Now acquire the current position from the reached `z_offset` using:
+Now, it's time to test homing with a twist. Instead of letting the probe pin touch the print bed,
+let it touch the nail on your finger. So issue a `G28`, wait until it starts to move down, and stop
+the movement by very gently touching the pin with your nail. You probably have to do it twice, since
+the default configuration makes it probe twice. But be prepared to turn off the printer, to avoid
+damage, if it doesn't stop when you touch the pin.
 
-    GET_POSITION
+If that was successful, do another `G28` but this time let it touch the bed as it should.
 
-In the output lines the `toolhead` line with the `Z:` contains the required value.
+Calibrating the BL-Touch
+========================
 
 ### X/Y Offset
-In order to find the propper vertical offset of the probe you need to know a certain point of
+
+In order to make klipper work properly you need to tell it in which relation to the nozzle
+the probe is exactly located. Lets start with the `x_offset` and the `y_offset`
+
+In order to find the proper vertical offset of the probe you need to know a certain point of
 your bed. X/Y = 0 may be a good point for this, any other will do as long as you know it.
-Find it by lowering the nozzle next to it using `g0 z0.5`. Create a removeable mark on your
+Find it by lowering the nozzle next to it using `g0 z0.5`. Create a removable mark on your
 bed by i.e. using a non permanent marker.
 
-Now move the tip of the BL-Touch over that point by jogging there with the conrols in the `control` tab of
+Now move the tip of the BL-Touch over that point by jogging there with the controls in the `control` tab of
 OctoPrint. Once the BL-Touch is roughly over the point acquire it using the `GET_POSITION` command.
 The difference to your marked point is your `x_offset` and `y_offset` to configure in the `printer.cfg`.
 
 
+### Z Offset
+
+1. We start by changing the `z_offset` in the `bltouch` section of the configuration to 10. This
+setting is very wrong, but it will make it possible to move down to the right height using the
+menus. Run `RESTART` after changing it to reload the configuration.
+
+2. Now run `G28`, to home the printer. Take a note of the point where the pin hits the bed and use
+the menu to move the nozzle to that point. Now take a *folded* paper and put it under the nozzle.
```

Comment:
Since we've already calculated our X axis, would it be accurate send a `G91` (relative positioning) and then move X your offset? (`G1 X[offset]`)  This should get you right to the spot your pin fired, and be a lot easier to understand for beginners.

Reply:
Yes, that's actually how I did it myself. I didn't describe it that way, because when I initially wrote that section, it came before calibrating the x/y offsets. But now when I changed the order, it would indeed be easier.

The only small complication with this, is that the positioning has to be set back to absolute, before doing something else, otherwise it could mess things up. It would be nice to have separate commands for absolute and relative movement for situations like these.

Reply:
Yup, I was also thinking that first measuring x/y offset would be the more clever approach, since one should probably `PROBE` on the same place where the nozzle was tested.